src/colors.css

```diff
@@ -124,15 +141,20 @@ html[data-color-scheme="dark-high-contrast"] {
   --color-neutral90: #fff;
 
   --color-danger0: #000;
-  --color-danger0-bw-iInverted: #fff;
+  --color-danger0-bw-inverted: #fff;
   --color-danger1: #000;
   --color-danger1-bw-inverted: #fff;
   --color-danger2: #eb1722;
+  --color-danger2-bw-inverted: #fff;
   --color-danger4: #eb1722;
+  --color-danger4-bw-inverted: #fff;
   --color-danger5: #eb1722;
+  --color-danger5-bw-inverted: #fff;
```

Comment:
Close call but I think #000 for these three. Tho I guess a "close call" is not acceptable in high contrast mode...  graa everything is so complicated. Maybe the danger color here is not great? I feel like it should be lighter. Maybe `#fb7c67` (which is `danger5` in non-high-contrast mode).

Reply:
Mmh yeah, we'll have to change the danger I suppose. I'll try some.